Patches/DefensiveMachineGunTurretPack/DMGT_CE_Patch_ThingDefs_Defensive_Turrets.xml

```diff
@@ -186,14 +186,14 @@
 					<recoilPattern>Mounted</recoilPattern>
 				</Properties>
 				<AmmoUser>
-					<magazineSize>500</magazineSize>
```

Comment:
As per [the catalog on the Dillon Aero website](https://dillonaero.com/wp-content/uploads/2016/07/7L-DADS0044-11-X-17-AMMUNITION-MAGAZINES-AND-FEED-SYSTEMS-FOR-GROUND_NAVAL-APPLICATIONS-03-26-18.pdf), there are definitely 3000~4000 round magazines specifically designed for the M134 minigun.

One of them has a NSN of [1005-01-512-4416](https://www.wbparts.com/rfq/1005-01-512-4416.html), and is explicitly stated for use with the **Machine Gun, 7.62mm, M134 (Mini-Gun)**.

Since I am also the current maintainer for the original DMGT mod, if necessary I can update the turret graphics to visually have a larger ammo can (although I doubt this would be necessary, since the 3000-round can is depicted as sitting on the ground while feeding a belt to the minigun).

Reply:
Those seem to me to be feed systems, rather than ammo cans, with the distinction that you can pretty quickly swap out an empty ammo can with a full one, while a feed system needs to be opened and refilled with rounds. Here's a picture of the 3000 round version from that catalogue--it's basically as large as the gun itself.
![Capture](https://user-images.githubusercontent.com/38633594/89926929-9275d680-dbd3-11ea-9eb2-5c5ddebba761.PNG)

Reply:
I couldn't find any real-life 1500-round ammo cans for the M134 (no real products, no NSNs, nothing), but I guess this will have to do for now.

Fixed in 1e29f8a